PhpAmqpLib/Message/AMQPMessage.php

```diff
@@ -39,9 +39,9 @@ class AMQPMessage extends GenericContent
 
     /**
      * @param string $body
-     * @param null $properties
+     * @param array $properties
      */
-    public function __construct($body = '', $properties = null)
+    public function __construct($body = '', $properties = array())
```

Comment:
I would prefer the empty check as well. I know this is PHP but `empty` is explicit, while an empty array assume things about PHP's behaviour